src/app.js

```diff
@@ -355,17 +378,25 @@ class Messenger extends EventEmitter {
     return this.cache.set(session._key, session);
   }
 
-  send(recipientId/*: string|number */, messageData/*: Object */, pageId/*: string|void */)/* Promise<Object> */ {
+  send(arg1/*: string|number */, arg2/*: string|number|Object */, arg3/*: Object|void */)/* Promise<Object> */ {
```

Comment:
I've been working on this concept on and off for a few weeks. I think I thought of that approach but I can't remember why I didn't stick with it. I'll give it a shot with `pageSend` 👍